scss/_typography.scss

```diff
@@ -36,16 +36,20 @@ p {
 a {
   color: $color__link-blue;
   text-decoration: underline;
+
   &:hover {
```

Comment:
I think the `:hover` pseudo class might need to go after `:visited` (we've experienced some bugs recently in KBS where the styles haven't followed the LVHA ordering https://developer.mozilla.org/en-US/docs/Web/CSS/:link)

Reply:
Thanks Gwyn. This is great. The hover and focus pseudos will be done separately and are not complete. I should have pointed this out to you. Also the content for the header and footer are just placeholders. I'm not concerned with this at this point because the IA will dictate this content and possibly how the header and footer behaves.

Reply:
OK. I'll merge the PR now since these issues are going to be picked up later.